src/transformers/models/swin/configuration_swin.py

```diff
@@ -145,3 +151,20 @@ def __init__(
         # we set the hidden_size attribute in order to make Swin work with VisionEncoderDecoderModel
         # this indicates the channel dimension after the last stage of the model
         self.hidden_size = int(embed_dim * 2 ** (len(depths) - 1))
+
+
+class SwinOnnxConfig(OnnxConfig):
+
+    torch_onnx_minimum_version = version.parse("1.11")
```

Comment:
Is there any reason you fixed a minimal version for this configuration?

Reply:
I don't know the basic requirement, it's the same in every other Vision Onnx config-  without this package.version importing styling error in other Vit, BEiT, data2vec. So I maintained the same.

Reply:
Ok, it makes sense!

Reply:
Yes, we found that the PyTorch ONNX exporter had various issues for `torch <= 1.10.x` which is why it's fixed this way for all image models :)